alibi/api/defaults.py

```diff
@@ -249,3 +249,22 @@
 """
 Default CounterfactualRL data.
 """
+
+# Similarity methods
+DEFAULT_META_SIM = {"name": None,
+                    "type": ["whitebox"],
+                    "explanations": ["local"],
+                    "params": {},
+                    "version": None}  # type: dict
+"""
+Default SimilarityExplainer metadata.
+"""
+
+DEFAULT_DATA_SIM = {"scores": None,
+                    "X_train": None,
+                    "Y_train": None,
```

Comment:
What are the reasons for returning the training data?

Reply:
The training data is ordered by similarity and returned? I've made it more explicit and called it `ordered_X_train` and `ordered_Y_train`? Alternatively, I could just return the ordered indices?

Reply:
I think returning just indices is a great idea as would reduce the return payload and not take up additional space for a copy of the training data, I would be in favour of making this change. @RobertSamoilescu keen to hear your thoughts too.

Reply:
Hmm I'm not a huge fan of complicating the public API and putting conditions on the output format based on the input arguments. I would say stick with indices for now and if there is a need we can extend the API later.

Reply:
Ok, will change to just indices!